src/compressionworker.ts

```diff
@@ -0,0 +1,97 @@
+import compress from "./compress";
+import { config } from "./config";
+
+export function createCompressionWorker(
+  sampleEnvelope: IEnvelope,
```

Comment:
>sampleEnvelope [](start = 2, length = 14)

nit: sample is a confusing descriptor for me here. Is initialEnvelope a more accurate name perhaps?

Reply:
It's really the same object, that we re-use by updating sequence number for each next message - i think even just 'envelope' would be an ok name

---
In reply to: [136672217](https://github.com/Microsoft/clarity-js/pull/28#discussion_r136672217) [](ancestors = 136672217)